packages/lib/src/ContentContainer.wc.svelte

```diff
@@ -28,6 +28,11 @@
   // TODO: This should be moved to sesamy-js
   async function getRemoteContent(api: SesamyAPI, entitlement: boolean | Entitlement) {
     if (!accessUrl) {
+      const contentSlot = $host().querySelector('div[slot="content"]');
```

Comment:
Ah, as it's the host it would start from the web component level?

Reply:
yes exactly